RIOT-OS/RIOT
# drivers/at86rf215: implement MR-O-QPSK

### Contribution description

This adds MR-O-QPSK modulation to the `at86rf215` driver.

This also adds new commands to `ifconfig` to configure the new modulation parameters.

### Testing procedure

Switch to the new PHY mode with `ifconfig 7 set phy_mode mr-o-qpsk`. Note that the `L2-PDU` has changed to 2022 bytes (802.15.4g can send 2047 byte frames).

You can now change the rate mode with `ifconfig 7 set rate_mode 3` (range: 0…4) and the chip rate with `ifconfig 7 set chips 1000`  (possible values: 100, 200, 1000 & 2000 chip/s)

### Issues/PRs references

split off #12128 but now the timings should be correct.
